localflavor/br/forms.py

```diff
@@ -147,34 +119,15 @@ class BRCNPJField(CharField):
 
     def __init__(self, min_length=14, max_length=18, *args, **kwargs):
         super(BRCNPJField, self).__init__(max_length=max_length, min_length=min_length, *args, **kwargs)
+        self.validators.append(validators.BRCNPJValidator())
 
     def clean(self, value):
         """Value can be either a string in the format XX.XXX.XXX/XXXX-XX or a group of 14 characters."""
         value = super(BRCNPJField, self).clean(value)
         if value in self.empty_values:
             return self.empty_value
-        orig_value = value[:]
-        if not value.isdigit():
-            cnpj = cnpj_digits_re.search(value)
-            if cnpj:
-                value = ''.join(cnpj.groups())
-            else:
-                raise ValidationError(self.error_messages['invalid'])
-
-        if len(value) != 14:
-            raise ValidationError(self.error_messages['max_digits'])
-        orig_dv = value[-2:]
-
-        new_1dv = sum([i * int(value[idx]) for idx, i in enumerate(list(range(5, 1, -1)) + list(range(9, 1, -1)))])
-        new_1dv = dv_maker(new_1dv % 11)
-        value = value[:-2] + str(new_1dv) + value[-1]
-        new_2dv = sum([i * int(value[idx]) for idx, i in enumerate(list(range(6, 1, -1)) + list(range(9, 1, -1)))])
-        new_2dv = dv_maker(new_2dv % 11)
-        value = value[:-1] + str(new_2dv)
-        if value[-2:] != orig_dv:
-            raise ValidationError(self.error_messages['invalid'])
-
-        return orig_value
+        original_value = value[:]
+        return original_value
```

Comment:
You should just do `return value` instead of these 2 lines. The same applies to the `BRCPFField`.